src/bsp/platform/cc3220/xi_bsp_io_net_cc3220.c

```diff
@@ -142,11 +155,24 @@ xi_bsp_io_net_connect( xi_bsp_socket_t* xi_socket, const char* host, uint16_t po
     }
 
     unsigned long uiIP;
-    int errval =
-        sl_NetAppDnsGetHostByName( ( _i8* )host, strlen( host ), &uiIP, SL_AF_INET );
+    int errval = 0;
+
+#ifdef XI_BSP_IO_NET_TLS_SOCKET
+    /* Set hostname for verification by the on-board TLS implementation */
```

Comment:
I'd put this setting next to other setsockopt calls into the function above: xi_bsp_io_net_create_socket
With this move there will be one less #ifdef XI_BSP_IO_NET_TLS_SOCKET compiler branching